infrastructure/aws/cdk/stac_browser_config.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+    catalogUrl: null,
+    catalogTitle: "eoAPI STAC Browser",
+    allowExternalAccess: true, // Must be true if catalogUrl is not given
+    allowedDomains: [],
+    detectLocaleFromBrowser: true,
+    storeLocale: true,
+    locale: "en",
+    fallbackLocale: "en",
+    supportedLocales: [
+        "de",
+        "es",
+        "en",
+        "fr",
+        "it",
+        "ro"
+    ],
+    apiCatalogPriority: null,
+    useTileLayerAsFallback: true,
+    displayGeoTiffByDefault: false,
+    buildTileUrlTemplate: ({href, asset}) => "https://raster.eoapi.dev/cog/tiles/{z}/{x}/{y}@2x?url=" + encodeURIComponent(asset.href.startsWith("/vsi") ? asset.href : href),
```

Comment:
same comment as for the other local deployment config (url hard coded but I have no quick solution for now)

Reply:
as usual I don't think we should have `https://raster.eoapi.dev` in this example of deployment, but I don't think there is a better solution :-(

Reply:
The only way I can think of to avoid this is the following, and it's a bit messy 

 - add a browser_tiler_url settings var 
 - in current `stac_browser_config.js`, remove the currently hard coded raster.eoapi.dev (replace an empty value that would trigger an error for example).

then in app.py, when we deploy the browser: 

- copy `stac_browser_config.js` to a temp file
- in that temp file, dynamically add the tiler url parameter.

Reply:
that's seems a bit complex! 

because it's a `.js` file maybe we can add env variable directly? 

Or I would do a 
- create a`stac_browser_config.js.example`
-  add `stac_browser_config.js` in .gitignore
- add note in documentation that user need to create a `stac_browser_config.js` file if they need to deploy the stac-browser

Reply:
> that's seems a bit complex!

totally, went for the `.example` way 👍